runtime/centrifuge/src/liquidity_pools.rs

```diff
@@ -0,0 +1,126 @@
+// Copyright 2023 Centrifuge Foundation (centrifuge.io).
+//
+// This file is part of the Centrifuge chain project.
+// Centrifuge is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version (see http://www.gnu.org/licenses).
+// Centrifuge is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+use cfg_primitives::{
+	liquidity_pools::GeneralCurrencyPrefix, AccountId, Balance, EnsureRootOr, PalletIndex, PoolId,
+	TrancheId, TwoThirdOfCouncil,
+};
+use cfg_types::{
+	fixed_point::Ratio,
+	tokens::{CurrencyId, TrancheCurrency},
+};
+use frame_support::{parameter_types, traits::PalletInfoAccess};
+use pallet_liquidity_pools::hooks::{
+	CollectedForeignInvestmentHook, CollectedForeignRedemptionHook, DecreasedForeignInvestOrderHook,
+};
+use runtime_common::{
+	account_conversion::AccountConverter, foreign_investments::IdentityPoolCurrencyConverter,
+	gateway::GatewayAccountProvider, liquidity_pools::LiquidityPoolsMessage,
+	origin::EnsureAccountOrRootOr,
+};
+use sp_runtime::traits::One;
+
+use crate::{
+	ForeignInvestments, Investments, LiquidityPools, LiquidityPoolsAxelarGateway,
+	LiquidityPoolsGateway, LocationToAccountId, OrderBook, OrmlAssetRegistry, Permissions,
+	PoolSystem, Runtime, RuntimeEvent, RuntimeOrigin, Timestamp, Tokens, TreasuryAccount,
+};
+
+parameter_types! {
+	pub DefaultTokenSellRatio: Ratio = Ratio::one();
+}
+
+impl pallet_foreign_investments::Config for Runtime {
+	type Balance = Balance;
+	type BalanceRatio = Ratio;
+	type CollectedForeignInvestmentHook = CollectedForeignInvestmentHook<Runtime>;
+	type CollectedForeignRedemptionHook = CollectedForeignRedemptionHook<Runtime>;
+	type CurrencyConverter = IdentityPoolCurrencyConverter<OrmlAssetRegistry>;
+	type CurrencyId = CurrencyId;
+	type DecreasedForeignInvestOrderHook = DecreasedForeignInvestOrderHook<Runtime>;
+	type DefaultTokenSellRatio = DefaultTokenSellRatio;
+	type Investment = Investments;
+	type InvestmentId = TrancheCurrency;
+	type PoolId = PoolId;
+	type PoolInspect = PoolSystem;
+	type RuntimeEvent = RuntimeEvent;
+	type TokenSwapOrderId = u64;
+	type TokenSwaps = OrderBook;
+	type TrancheId = TrancheId;
+	type WeightInfo = ();
+}
+
+parameter_types! {
+	// To be used if we want to register a particular asset in the chain spec, when running the chain locally.
+	pub LiquidityPoolsPalletIndex: PalletIndex = <LiquidityPools as PalletInfoAccess>::index() as u8;
+}
+
+impl pallet_liquidity_pools::Config for Runtime {
+	// NOTE: No need to adapt that. The Router is an artifact and will be removed
```

Comment:
Artifact?

Reply:
Artifact indeed, sorry about that. Can I remove this in a post-release PR? CI takes ages as each push builds new images. Next release I need to apply another branching strategy, i.e. have a `release candidate` branch which wants to merge into `release-vx.y.z` which is based on `main` after feature freeze. This way, we don't need to rebase the `rc` branch on pushes into main as well.